Tasks/Common/azure-arm-rest/azureModels.ts

```diff
@@ -213,6 +215,7 @@ export interface AzureEndpoint {
     portalEndpoint?: string;
     AzureKeyVaultDnsSuffix?: string;
     AzureKeyVaultServiceEndpointResourceId?: string;
+    applicationTokenCredentials: ApplicationTokenCredentials;
```

Comment:
Should this be a method here instead given this is derived value? like getArmTokenCredentials

Reply:
Planned to have attributes here instead of methods. 
Also, it tends to honour as an attribute for AzureEndpoint.